app/modes/library/routes.py

```
@@ -0,0 +1,769 @@
+
+
+
+# Global dictionary to track generation progress for books - DEPRECATED
+# Now using BookGenerationProgress model in database for persistent tracking
+active_generations = {}
+
+def generate_book_background(app_context, book_id, user_background, user_id):
+    """Background task to generate all content for a book - DEPRECATED, use generation.py"""
+    # This function is kept for backwards compatibility but should not be used
+    # Use app.modes.library.agent.generate_book_content_background instead
+    logger.warning(f"Using deprecated generate_book_background for book {book_id}")
```

Ruff (0.15.1)

[warning] Logging statement uses f-string

(G004)